torch/onnx/_internal/exporter/_compat.py

```diff
@@ -98,25 +101,18 @@ def _from_dynamic_axes_to_dynamic_shapes(
 
     dynamic_shapes: dict[str, Any | None] = {}
     for input_name, axes in dynamic_axes.items():
-        # NOTE: torch.export.Dim requires strict min and max constraints, and it
-        # dpends on the traced model to provide the correct min and max values.
-        # We set max to 99999 to avoid the constraints violation error with the default int64 max.
-        # https://github.com/pytorch/pytorch/blob/32f585d9346e316e554c8d9bf7548af9f62141fc/test/export/test_export.py#L687
+        # TODO(titaiwang): Add ONNX IR pass to post-process the dynamic axes
```

Comment:
Is it possible to include the information returned by AUTO dim analysis in the export-report? For example, if analysis indicates dim1 and dim2 are expected to be equal? Or, if it indicates dim1 is expected to be in range (5, 10)?

Reply:
This can a follow-up PR that we try to identify the constraints and store them in metadata of ONNX IR.